### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by mohdzakwan99**
ready

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

**Comment by mohdzakwan99** [comment]
For the 2 commits, i tried to pull from chapter_6 into local and squash it using git rebase -i HEAD~N and then after i finally solved all the conflicts and so on during the squash i push again to branch chapter_6 in git and there were conflicts (even though the contents were the same with local), so after that i pull again from chapter_6 and solved all the conflicts but then there were multiple commits after i solved the conflicts in branch chapter_6. After that i did the thing from the start to squash the commits and then push again to github only to get the same conflicts and then after resolving that I will get multiple commits again. I tried many ways to squash the commits since i have so many commits in the branch chapter_6 but i did not succeed.

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,7 @@
 class User < ApplicationRecord
-<<<<<<< HEAD
-=======
   before_save { self.email = email.downcase }
   validates :name, presence: true
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
   validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }
   has_secure_password
```
Comment (huongnt-2545) [inline app/models/user.rb:6]: You seem missing validate password, please additional after this line
```
validates :password, presence: true, length: { minimum: 6 }
```

**Comment by mohdzakwan99** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**